djangocms_versioning/static/djangocms_versioning/js/compare.js

```diff
@@ -0,0 +1,14 @@
+window.addEventListener('load', function() {
+  document.querySelector('select[name=action]').addEventListener('change', function() {
+    var selectVal = this.options[this.selectedIndex].text;
+    if (selectVal == "Compare versions"){
```

Comment:
Wondering if this will work when the user has changed the language where "Change versions will be different.

Reply:
@Aiky30 Good point. Tested this and the action dropdowns to not translate but the Go button will be translated. Looking at another way of checking this now

Reply:
@Bernardvdv The Compare Versions won't be translated yet because no translations have been submitted for versioning strings. One day soon it will start to receive translations.